src/App.js

```diff
@@ -457,7 +479,12 @@ function App() {
             setCity={setCity}
             setInput={setInputValue}
           />
+
+          {data!==undefined && data!==null && results!==undefined && results!== null &&
+            <ForecastCard data={data} results={results} /> 
+          }
           <br />
+
         </span>
         {data !== undefined &&
```

Comment:
Hi! Please delete from line 489 to line 497 so that the forecast cards don't repeat.